pkg/helper/versions/helpers.go

```diff
@@ -114,3 +114,23 @@ func IsGreaterThanOrEqual(version1, version2 string) (bool, error) {
 	}
 	return v1.GreaterThanOrEqual(v2), nil
 }
+
+func FormatMajorMinorPatch(version string) (string, error) {
+	major, minor, patch, err := getVersionSegments(version)
+	if err != nil {
+		return "", err
+	}
+	return fmt.Sprintf("%d.%d.%d", major, minor, patch), nil
+}
+
+func getVersionSegments(rawVersionID string) (major, minor, patch int, err error) {
+	v, err := ver.NewVersion(rawVersionID)
```

Comment:
NP: I think elad has brought up before that we should be verbose when possible with variable names -- not sure if it matters here as this is a simple helper function, but going through other helper functions in our codebase it seems we generally use full names rather than single letters